nomad/structs/structs.go

```diff
@@ -97,6 +97,8 @@ const (
 	ScalingEventRegisterRequestType
 	CSIVolumeClaimBatchRequestType
 	CSIPluginDeleteRequestType
+	EventSinkUpsertRequestType
+	EventSinkDeleteRequestType
```

Comment:
Oof I'm going to have caused you a merge conflict with this and raftutil msgtypes in #9135 

Sorry!